Cinema/src/main/java/com/example/cinema/model/requestModel/MovieUpdateReq.java

```diff
@@ -26,6 +26,15 @@
     @ValidTime(groups = ThirdPriorGroup.class)
     private LocalTime time;
 
+    public MovieUpdateReq(Integer id, String name, @Min(value = 1, groups = SecondPriorGroup.class) @Max(value = 10, groups = SecondPriorGroup.class) Double grade, @Min(value = 1900, groups = SecondPriorGroup.class) @Max(value = 2021, groups = SecondPriorGroup.class) Integer year, Integer idGenre, LocalTime time) {
+        this.id = id;
+        this.name = name;
+        this.grade = grade;
+        this.year = year;
+        this.idGenre = idGenre;
+        this.time = time;
+    }
+
```

Comment:
Constructor for purpose of testing

Reply:
Try to avoid adding constructors and methods, if they are only used to enable unit tests. Same thing applies to access modifiers (public, private, protected), try to avoid changing them just because of unit tests.

Reply:
All constructors are now zero argument constructor. For purpose of program I didn't need any other constructor. Fixed